utils/m0reportbug

```diff
@@ -645,11 +645,6 @@ backtraces_cores() {
             cp $core $outdir/$IN_DIR/
         fi
     done
-
-    if [ -d $outdir ] && ! rmdir $outdir 2>/dev/null; then
-        ## Core dumps were collected. We need to collect binaries.
-        COLLECT_BINARIES_P=1
-    fi
```

Comment:
Removing this part will not collect binaries if core are  there and --binaries is not set to true. 

I think we should maintain existing behavior of "m0reportbug" script and do necessary changes at in wrapper script "m0reportbug-bundler"

Reply:
created jira to address this issue
https://jts.seagate.com/browse/EOS-26951